zonemaster/zonemaster-backend
# Fixes broken unit tests

## Purpose

Unit test file `t/test01.t` leaks queries when network is set to off. A change in .fr NS and DS triggered the tests to break. This PR:

* Updates `t/test01.t` with correct expectations on .fr.
* Updates `t/test01.t` so that it does not leak queries.
* Updates `t/test01.data` to match updates on .fr and updated `t` file.
* Updates `t/idn.t` so that it does not leak queries.

## Context

Issue #1191

## How to test this PR

1. Unit tests must pass.
2. Review.
3. Verify that no queries are sent when network is set to off.
